thorin/world.h

```diff
@@ -568,6 +572,7 @@ class World {
         swap(w1.err_,      w2.err_);
         // clang-format on
 
+        swap(w1.data_.imported_dialects_, w2.data_.imported_dialects_);
```

Comment:
shouldn't this be covered by the
```c++
swap(w1.data_,     w2.data_);
```
above?

Reply:
should be fixed. The field should be copied upon `World::stub`.